### DISCUSSION TITLE
Drum fills with Hydrogen

### DISCUSSION THREAD

**Discussion by vinodpkd** [hydrogen-music/hydrogen#1998] [Show and tell]
Hello friends,
I have made drum fill patterns in Hydrogen as per the patterns  given in the youtube video : https://www.youtube.com/watch?v=cPlcYbH676Y

https://github.com/vinodpkd/drumpatterns/blob/main/Drum_fills_Hydrogen.zip

While converting to wave file, I have observed 1 second silence in the end. How to get rid of this silence.

If you like please star my github repo.

I have uploaded a video for the drum fills made with Hydrogen
https://www.youtube.com/watch?v=3kByrUwjG7k

**Comment by theGreatWhiteShark** [comment]
Hey @vinodpkd 

Thanks for sharing!

I noted that you stored those patterns as `.h2song` files. Those carry a bunch of additional information not required for the pattern itself. You are also able to save/export a pattern as `.h2pattern` file which just contains the info required.
